File: swap.py
import os

path = r"C:\Users\LENOVO\.gemini\antigravity\scratch\index.html"
with open(path, 'r', encoding='utf-8') as f:
    lines = f.readlines()

brands_start = -1
products_start = -1
features_start = -1

for i, line in enumerate(lines):
    if '<!-- Brands Section -->' in line:
        brands_start = i
    elif '<!-- Products Section (Sebang Style) -->' in line:
        products_start = i
    elif '<!-- Features Section -->' in line:
        features_start = i

if brands_start != -1 and products_start != -1 and features_start != -1:
    before = lines[:brands_start]
    brands = lines[brands_start:products_start]
    products = lines[products_start:features_start]
    after = lines[features_start:]

    new_lines = before + products + brands + after
    with open(path, 'w', encoding='utf-8') as f:
        f.writelines(new_lines)
    print('Swapped successfully.')
else:
    print(f'Error finding sections: brands_start={brands_start}, products_start={products_start}, features_start={features_start}')

File: swap.js
const fs = require('fs');

const path = 'C:\\Users\\LENOVO\\.gemini\\antigravity\\scratch\\index.html';
const content = fs.readFileSync(path, 'utf8');
const lines = content.split('\n');

let brands_start = -1;
let products_start = -1;
let features_start = -1;

for (let i = 0; i < lines.length; i++) {
    if (lines[i].includes('<!-- Brands Section -->')) {
        brands_start = i;
    } else if (lines[i].includes('<!-- Products Section (Sebang Style) -->')) {
        products_start = i;
    } else if (lines[i].includes('<!-- Features Section -->')) {
        features_start = i;
    }
}

if (brands_start !== -1 && products_start !== -1 && features_start !== -1) {
    const before = lines.slice(0, brands_start);
    const brands = lines.slice(brands_start, products_start);
    const products = lines.slice(products_start, features_start);
    const after = lines.slice(features_start);

    const new_lines = [].concat(before, products, brands, after);
    fs.writeFileSync(path, new_lines.join('\n'), 'utf8');
    console.log('Swapped successfully.');
} else {
    console.log(`Error: brands_start=${brands_start}, products_start=${products_start}, features_start=${features_start}`);
}
